src/dbt_score/cli.py

```diff
@@ -93,8 +93,17 @@ def cli() -> None:
     is_flag=False,
     default=None,
 )
+@click.option(
+    "--show_all",
```

Comment:
```suggestion
    "--show-all",
```
Let's use a dash for the CLI

Reply:
Implemented!